### PR TITLE
fix(app): #45 Change path for zipfile export

### PR DESCRIPTION
**Pull request by AlexAxthelm**
Change path for zipfile export. Since the code changes the working directory to `output_dir`, there is no need to construct a filepath without `output_dir` in it, since we can simply export to a simple file in the (now) working directory.

Prior to this change, zip would attempt to write to `output_dir/output_dir/foo.zip`, and now it writes to the path defined in the original `zipfilename`

Closes: #45

### CODE REVIEW

**Review by cjyetman — APPROVED** [review]
This makes sense to me, but obviously there are implications to this on both TM and the new Webapp which I think are very hard to discern. Also, probably @MonikaFu (as the maintainer) should approve this.

**Comment by AlexAxthelm** [comment]
> there are implications to this on both TM and the new Webapp

To my knowledge, CTM doesn't use the zip files in any way

**Comment by cjyetman** [comment]
> > there are implications to this on both TM and the new Webapp
> 
> To my knowledge, CTM doesn't use the zip files in any way

True, but the code runs regardless

**Pull request merged**
